src/utils/get-transfer-data.js

```diff
@@ -81,6 +81,61 @@ function getTransferType(data) {
 }
 
 /**
+ * Get fragment from transfers's `data` if possible, otherwise return null
+ *
+ * @param {Object} transfer
+ * @return {String}
+ */
+
+function getFragment(transfer) {
+  return transfer.types && transfer.types.indexOf(TYPES.FRAGMENT) !== -1 ? transfer.getData(TYPES.FRAGMENT) || null : null
+}
+
+/**
+ * Get node from transfers's `data` if possible, otherwise return null
+ *
+ * @param {Object} transfer
+ * @return {String}
+ */
+
+function getNode(transfer) {
+  return transfer.types && transfer.types.indexOf(TYPES.NODE) !== -1 ? transfer.getData(TYPES.NODE) || null : null
+}
+
+/**
+ * Get html from transfers's `data` if possible, otherwise return null
+ *
+ * @param {Object} transfer
+ * @return {String}
+ */
+
+function getHtml(transfer) {
+  return transfer.types && transfer.types.length ? transfer.getData('text/html') || null : null
```

Comment:
Is there a reason that these ones use `&& transfer.types.length` and the ones above use `&& transform.types.indexOf(type)`?